docs/en/rs-school-trainer.md

```diff
@@ -1,36 +1,45 @@
-•	If you specify any details of the task, please do not forget to add these details to the description, not just drop them into the general chat.
+
+- Deadline of tasks in the format - 11:59 p.m. 03/03/2016. Deadlines are only on Tuesdays, Thursdays, Saturdays and Sundays, that is, on days when there are no lectures.
+- Tasks must be added to the course schedule.
+- Tasks should be well-defined (details on browser support, what and how to resize, scroll, etc.).
+- Information on how to hand over the task (where and how to upload it, etc.).
+- If you specify any details of the task, please do not forget to add these details to the description, not just drop them into the general chat.
 
 ### Theoretical tasks requirements
-•	No more than 2 hours of theory for each lesson. Theoretical knowledge should be covered by practice - task, test or other activity. Everything that is put into practice is quickly forgotten.
-•	Offline time should include 1-2 breaks
-•	Remove all unnecessary information from presentations. All the main links that can be  useful for further reading on the topic should be given at the end of the presentation
-•	Before giving the presentation, omit all materials / slides that are out of date. For example, nobody is interested in layout features for IE6 or handling memory leaks in IE8
-•	The turned on webcam at webinars makes the lecture more cordial
-•	During offline classes, communicate with the entire audience, and not with the front rows
-•	It is recommended to use contrasting colors in the presentation, as all lectures are shown to the audience on the projector
-•	For specific and rarely used things, it is better to leave a link to the material, do not describe them in detail
-•	Make more simple slides rather than one huge
-•	To present an API analysis of any functionality, it is better to organize workshops, not just enumerate them
+
+- No more than 2 hours of theory for each lesson. Theoretical knowledge should be covered by practice - task, test or other activity. Everything that is put into practice is quickly forgotten.
+- Offline time should include 1-2 breaks
+- Remove all unnecessary information from presentations. All the main links that can be  useful for further reading on the topic should be given at the end of the presentation
+- Before giving the presentation, omit all materials / slides that are out of date. For example, nobody is interested in layout features for IE6 or handling memory leaks in IE8
+- The turned on webcam at webinars makes the lecture more cordial
+- During offline classes, communicate with the entire audience, and not with the front rows
+- It is recommended to use contrasting colors in the presentation, as all lectures are shown to the audience on the projector
+- For specific and rarely used things, it is better to leave a link to the material, do not describe them in detail
+- Make more simple slides rather than one huge
```

Comment:
Split slides into smaller parts rather than having all information in one huge slide